enterprise/suppress-user/model/model.go

```diff
@@ -9,6 +9,8 @@ var (
 	ErrRestoring    = errors.New("repository is restoring")
 	ErrNotSupported = errors.New("operation not supported")
 	SyncDoneMarker  = ".sync_done"
+	ErrNoValueFound = errors.New("no value found")
+	ErrKeyNotFound  = errors.New("key not found")
```

Comment:
I guess ErrKeyNotFound should be sufficient here. For both the implementation, we are looking for some key, if not present we are rerurning error.
```suggestion
	ErrKeyNotFound  = errors.New("key not found")
```